src/Core/src/HotReload/IHotReloadableView.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using Microsoft.Maui.Hosting;
+
+namespace Microsoft.Maui.HotReload
+{
+	public interface IHotReloadableView : IReplaceableView, IView
```

Comment:
an alternative to this would be to use a serialisation/deserialization mechanism